.github/workflows/uploadReleaseArtifacts.yml

```diff
@@ -13,13 +13,18 @@ on:
           - linux
           - macos
           - windows
+
+      # The following options only influence workflow_dispatch, and are ignored otherwise.
       runTests:
-        type: choice
         description: Run CIRCT tests
         default: false
-        options:
-          - true
-          - false
+        type: boolean
+
+      enableAssertsAndDebugInfo:
+        description: Build with assertions and debug information.
+        default: false
+        type: boolean
```

Comment:
It may be better to factor this into enabling asserts and the release/debug/relwithdebinfo version.

Reply:
Sounds good, updated! :+1: